components/SummaryBox/SummaryBox.tsx

```diff
@@ -1,11 +1,17 @@
-import { FC, useState, useEffect } from 'react';
 import { FormikProps } from 'formik';
-
 import styled from 'styled-components';
+import { FC, useState, useEffect } from 'react';
+
 import ConfirmTransactionModal from 'components/ConfirmTransactionModal';
+import Button from 'components/Button';
+
 import Connector from 'containers/Connector';
+
 import { GasLimitEstimate } from 'constants/networks';
 import { TransactionStatus } from 'constants/transactions';
+import { Privacy } from 'constants/pool';
+import WhiteList from 'components/WhiteList';
+import { IWhitelist } from 'components/WhiteList/types';
```

Comment:
What does the I for IWhitelist stand for?

Reply:
It means Interface but it could be WhitelistProps. I think the second one is better. Will change it.

Reply:
Done